taxcalc/decorators.py

```diff
@@ -283,7 +285,7 @@ def make_wrapper(func):
         try:
             jit_args = inspect.getargspec(jit).args + ['nopython']
         except TypeError:
-            print ("This should only be seen in RTD, if not install numba!")
+            #print ("This should only be seen in RTD, if not install numba!")
```

Comment:
I would definitely advocate for something other than just commenting out the print statement. One way to handle this is to attempt the import of `jit` at the top and if `numba` is not found, produce a dummy function. Something like this:

```
# Numba is not available when building on ReadTheDocs
try:
    from numba import jit
except ImportError:
    jit = lambda x: return x
```
then this line inside the function would simply be:

```
    jit_args = inspect.getargspec(jit).args + ['nopython']
```
because `jit` is defined in either case. How does that sound?